source/endpoints/series.md.erb

```diff
@@ -0,0 +1,291 @@
+
+Parameter | Description
+--------- | -----------
+url_ids   | Array of URL ids
+with_competitors | Include competititor URL series (true or false, default: true)
+date_from* | Start date for series in YYYY-MM-DD format (e.g 2019-01-02)
+date_to*   | End date for series in YYYY-MM-DD format (e.g. 2019-04-02)
+
+Parameters marked with * are mandatory.
+
+Available URL series:
+
+Series | Description
+--------- | -----------
+average_position    | Average rank position
+search_visibility_index   | Search visibility index ([more info](https://docs.nightwatch.io/article/51-metrics-explanation))
+click_potential | Click potential ([more info](https://docs.nightwatch.io/article/51-metrics-explanation))
+keyword_distribution | Distribution of keyword positions in Top 3, Top 10, Top 100, No rank
+total_keywords | Number of keywords tracked
+indexed_pages | Number of indexed pages on Google
+moz_domain_authority | Moz's Domain Authority
+moz_page_authority | Moz's Page Authority
+ahrefs_domain_rating | Ahrefs' Domain Rating
+ahrefs_rank | Ahrefs' URL Rank
+majestic_trust_flow | Majestic's Trust Flow
+majestic_citation_flow | Majestic's Citation Flow
+clicks | Number of organic search clicks (when Search Console property is connected)
+impressions | Number of organic search impressions (when Search Console property is connected)
+sessions | Number of Google Analytics Sessions (when Google Analytics property is connected)
+pageviews | Number of Google Analytics Pageviews (when Google Analytics property is connected)
+users | Number of Google Analytics Users (when Google Analytics property is connected)
+search_console_position | Search Console average position (when Search Console property is connected)
+ctr | Search Console CTR (when Search Console property is connected)
+
+## Keyword series
+
+```shell
+curl 'https://api.nightwatch.io/api/v1/series?access_token=ACCESS_TOKEN&keyword_ids%5B%5D=1212189&date_from=2019-06-04&date_to=2019-07-04'
+```
+
+> The above command returns JSON structured like this:
+
+```json
+[{
+  keyword_id: 1212189,
+  keyword: "goji berries",
+  keyword_series: [
+    {
+      name: "position",
+      series:  [["2019-04-22", 3], ...]
+    }, ...
+  ],
+  competitor_series: [
+    ...
+  ]
+]
+```
+
+### HTTP Request
+
+`GET https://api.nightwatch.io/api/v1/series?access_token=ACCESS_TOKEN`
+
+Parameter | Description
+--------- | -----------
+keyword_ids  | Array of keyword ids
+date_from* | Start date for series in YYYY-MM-DD format (e.g 2019-01-02)
+date_to*   | End date for series in YYYY-MM-DD format (e.g. 2019-04-02)
+
+Parameters marked with * are mandatory.
+
+Available keyword series:
+
+Series | Description
+--------- | -----------
+position  | Keyword's rank
+position_organic | Keyword's organic rank
+position_local_pack | Keyword's local pack rank
+position_places | Keywords rank in Google Places (when engine is Google Places)
+position_places_image | Keywords rank in image carousel (when engine is Google)
+
```

Comment:
Google Analytics series are missing:
 - `clicks`
 - `impressions`
 - `search_console_position`
 - `ctr`

Might be worth noting that if GA integration is not setup these series will be empty (`[]`). Same notice applies to URL series.